templates/systems/element_detail_tabs.html

```diff
@@ -87,176 +87,204 @@
-                    <option value='Unknown' {% if 'Unknown' == smt.status %}selected="selected"{% endif %}>Unknown</option>
-                    </select>
+      <div id="tab-content" class="row">
+          <!-- Nav tabs -->
+          <ul class="nav nav-tabs" role="tablist">
+              <li role="presentation" class="active"><a href="#component" aria-controls="component" role="tab" data-toggle="tab"><span class="glyphicon glyphicon-list-alt"></span> Component</a></li>
+
+              <li role="presentation"><a href="#component_controls" aria-controls="component_controls" role="tab" data-toggle="tab"><span class="glyphicon glyphicon-wrench"></span> Component Statements  &nbsp;<div id="common-tab-count">{{ impl_smts|length }}</div></a></li>
+              {% if enable_experimental_oscal %}
+                <li role="presentation"><a href="#oscal" aria-controls="oscal" role="tab" data-toggle="tab"><span class="glyphicon glyphicon-file"></span> OSCAL </a></li>
+
+              {% endif %}
+              {% if enable_experimental_opencontrol %}
+              <li role="presentation"><a href="#opencontrol" aria-controls="opencontrol" role="tab" data-toggle="tab"><span class="glyphicon glyphicon-file"></span> OpenControl </a></li>
+              {% endif %}
+          </ul>
+
+          <!-- Tab panels -->
+          <div class="tab-content">
+              <!-- Tab panel: component -->
+              <div role="tabpanel" class="tab-pane active" id="component">
+                  <div class="row">
+                      <div class="col-xs-10 col-sm-10 col-md-10 col-lg-10 col-xl-10">
+                          <h3>Description</h3>
+                          {% if element.description %}
+                            {{ element.description }}
+                          {% else %}
+                            No description exists for this component.
+                          {% endif %}
+                      </div>
+                      <div class="col-xs-2 col-sm-2 col-md-2 col-lg-2 col-xl-2">
+                          <div>&nbsp;</div>
+                      </div>
                   </div>
-                  <div class="form-group">
-                    <label for="remarks">Remarks</label>
-                    <textarea class="" class="form-control systems-element-form-remarks" id="remarks_{{ forloop.counter }}" name="remarks" placeholder="Add remarks for team" cols="50">{{ smt.remarks }}</textarea>
+              </div>
+              <!-- Tab panel: combined -->
+              <!-- Combined control content does not appear on this page -->
+              <!-- Tab panel: component_controls -->
+              <div role="tabpanel" class="tab-pane" id="component_controls">
+                  <div id="control-description" class="control-text"><h3>Component Implementations Statements</h3></div>
+                  <div id="smt-list" class="systems-smt-list">
+                      <!-- Loop through existing component-control statements -->
```

Comment:
Most of this file could be formatted for easier reading

Reply:
Format is the way it is in order to make files more comparable with similar edit component control implementation statement pages.